daphne_worker/src/lib.rs

```diff
@@ -319,19 +296,99 @@ impl DaphneWorkerRouter {
                     })
                     .get_async(
                         "/:version/collect/task/:task_id/req/:collect_id",
-                        |_req, ctx| async move {
-                            let task_id = parse_id!(ctx.param("task_id"));
-                            let collect_id = parse_id!(ctx.param("collect_id"));
+                        |req, ctx| async move {
```

Comment:
I don't think we want to do this, was we'd lose all of the draft02 tests that are testing something else, but use collection as part of the tests, e.g. that we got the right count of uploaded reports.  I haven't actually looked at how much would break yet, but my intuition is it is a lot!

Reply:
Sorry, I confounded this question with the larger issue of doing collection at all for draft02.  I guess we could change the code to just always POST instead of GET.   But since the code that is there works and follows draft02 still, I think I'd rather keep it and rip out all of draft02 in the future rather than write more code to have something that isn't quite draft02.

Reply:
Works for me, just thought I'd offer in case you were hoping to rip this out :)